app/views/events/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="container">
+  <h1 class="text-center pt-3 pb-2">Event</h1>
+
+  <div class="jumbotron jumbotron-fluid">
+    <div class="container">
+      <h1 class="display-4"><%= @event.name %></h1>
+      <p class="lead"><span class="text-white bg-secondary p-1">Details:</span> <%= @event.details %></p>
+      <p class="lead"><span class="text-white bg-secondary p-1">Location:</span> <%= @event.location %></p>
+      <p class="lead"><span class="text-white bg-secondary p-1">Date:</span> <%= @event.date %></p>
+      <p class="lead"><span class="text-white bg-secondary p-1">Host:</span> <%= @event.user.full_name %></p>
+
+      <% if current_user && current_user.id != @event.user_id && @event.date > DateTime.now %>
+        <% if current_user.attendees.find_by(event_id: @event.id) %>
+          <%= link_to "Delete", event_attendee_path(@event), method: :delete, id: "event_#{@event.id}", class: 'btn btn-danger' %>
+        <% else %>
+          <%= link_to "Join", event_attendees_path(@event, @attendee), method: :post, id: "event_#{@event.id}", class: "btn btn-primary" %>
+        <% end %>
```

Comment:
- [ ] Create a helper method and render the markup within the helper method instead of doing it straight in the view